cafebabel/templates/_social-networks-macro.html

```diff
@@ -3,6 +3,7 @@
     <li><a href={{ social_network_url_for('facebook') }} class=icon-facebook></a></li>
     <li><a href={{ social_network_url_for('twitter') }} class=icon-twitter></a></li>
     <li><a href={{ social_network_url_for('instagram') }} class=icon-instagram></a></li>
+    <li><a href=#static-pages-links class=icon-newsletter></a></li>
```

Comment:
Funny I hadn't notify this!
A link with no content is totally inaccessible (I mean all these social items).
We should have text in HTML so it's readable for all reading systems, and remove it from display on screens.
Technically that mean the HTML is like 
```html
<li><a href={{ social_network_url_for('twitter') }} class=icon-twitter>Cafebabel on Twitter</a></li>
```
and a CSS trick would be:
```css
nav#socials > a {
  text-indent: -9999%;
}
```

Reply:
hum ok I get your point but it doesn't work your way. If I add text content, I can't have them hidden so far. Any ideas?

Reply:
I did that cause I couldn't have it done fast enough with `text-indent: 9999px blabla` 😞

Reply:
I guess the title fallback could be a minimum with zero effort then.

Reply:
ok !!